lib/prompt.js

```diff
@@ -416,9 +416,11 @@ prompt.getInput = function (prop, callback) {
   schema = convert(schema);
   defaultLine = schema.default;
   name = prop.description || schema.description || propName;
-  raw = prompt.colors
-    ? [prompt.message, delim + name.grey, delim.grey]
-    : [prompt.message, delim + name, delim];
+  // only show the message delimiter if there is actually a message
+  var prefix = prompt.message && prompt.message.length > 0 ? prompt.message + delim : '',
+    raw = prompt.colors
+      ? [prefix, name.grey, delim.grey]
+      : [prefix, name, delim];
```

Comment:
This functionality has been fixed in the created-before-this-but-only-recently-merged #61